### ISSUE TITLE
Add support for Postgres + Pgvector

### ISSUE THREAD

**Issue by agamm** [llm-tools/embedJs#106]
Many real world use cases require robust DBs like postgres. With pgvector it can function as a vector db for a RAG.

Are there any plans to add postgres?

**Comment by adhityan** [comment]
Yes there are. It's next on the list of databases to add support. I can't give an exact date when this will be available, but it is likely to be merged by October.

**Comment by subtubes-io** [comment]
hello, checking in on this feature. Happy to help with the integration.

**Comment by mohamedsahel** [comment]
Hey, any updates about this feature?
